### ISSUE TITLE
Add apostrophe to St. John's 

### ISSUE THREAD

**Issue by f-hwang** [ASSETS-Conference/assets2024#77]
on the landing page, in the logo at the bottom, st. john's is missing an apostrophe

<img width="303" alt="image" src="https://github.com/ASSETS-Conference/assets2024/assets/50186727/23a1a099-309e-4bc6-8e86-645bdb291da9">

**Comment by surajgoraya** [comment]
Yep noted! I will have to manually re-export the asset files for the logos, this should be done soon. 

I will also likely readjust the logo to say "N.L" or "Newfoundland and Labrador", which may require a bit of tweaking of the logo layout

**Comment by f-hwang** [comment]
thanks. i think it would normally be NL (no dots) but also this abbreviation may not be widely understood, so if there's a way to make it work, spelling it out in full may be better.
